packages/tpetra/core/src/Tpetra_Distributor.cpp

```diff
@@ -159,6 +160,8 @@ namespace Tpetra {
         (out_.is_null (), std::logic_error, "Tpetra::Distributor::init: debug_ "
```

Comment:
Line 158: `if (debug_)` needs to read `if (verbose)`.  Ditto for the error message.  The issue is that the error messages need to print based on `verbose`.

Reply:
^^^ this is the only actual bug I found ^^^